runtime/bindings/python/iree/runtime/system_api.py

```diff
@@ -1,4 +1,3 @@
-# Lint as: python3
```

Comment:
https://github.com/google/iree/pull/9041 fixes the pytype error in this file:

`File "/home/runner/work/iree/iree/runtime/bindings/python/iree/runtime/system_api.py", line 291, in instance: No attribute '_instance' on SystemContext [attribute-error]`
https://github.com/google/iree/runs/6276554487?check_suite_focus=true